modules/displaySection.js

```diff
@@ -0,0 +1,45 @@
+const sections = document.querySelectorAll('section');
+const pageTitle = document.querySelector('#page-title');
+// Menu Links
+const listLink = document.querySelector('#m-list a');
+const addLink = document.querySelector('#m-add a');
+const contactLink = document.querySelector('#m-contact a');
+
+function displaySection(sectionToDisp) {
+  sections.forEach((section) => {
```

Comment:
- [ ] This function still needs to be refactored as well, please convert it to an arrow function.